docs/ci-setup.md

```diff
@@ -48,14 +48,14 @@ equinix_metal_project_id = "_redacted_"
 public_key_path = "~/.ssh/id_rsa.pub"
 
 # cluster location
-facility = "am6"
+metro = "da"
 
 # standalone etcds
 number_of_etcd = 0
 
 plan_etcd = "t1.small.x86"
 
-# masters
+# masters                                                                                                             
```

Comment:
```suggestion
# masters
```
Spacing in these changes is offending the linter. It's hard to tell from the GitHub view, but it appears as though there are hundreds of spaces after the word `masters`.

Please do the following to test this locally.

```
$ npm install -g markdownlint-cli@0.22.0
added 36 packages in 2s
2 packages are looking for funding
  run `npm fund` for details
$ markdownlint $(find . -name '*.md' | grep -vF './.git') --ignore docs/_sidebar.md --ignore contrib/dind/README.md
./docs/ci-setup.md:58:10 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 109]
```

Reply:
Or two separate commits, one for the UA prefix, and another for the facility to metro changes.

The PR description should mention why this is being done: 
https://feedback.equinixmetal.com/changelog/bye-facilities-hello-again-metros